src/components/FlawTrackers.vue

```diff
@@ -0,0 +1,460 @@
+
+type TrackerWithModule = ZodTrackerType & { ps_module: string };
+
+const props = defineProps<{
+  flawId: string;
+  displayedTrackers: TrackerWithModule[];
+  affectsNotBeingDeleted: ZodAffectType[];
+  allTrackersCount: number;
+}>();
+
+const emit = defineEmits<{
+  'file-tracker': [value: object];
+  'affects:refresh': [];
+}>();
+
+const settingsStore = useSettingsStore();
+const settings = ref(settingsStore.settings);
+
+const showTrackerManager = ref(false);
+
+const hasTrackers = computed(() => props.allTrackersCount > 0);
+
+// Sorting
+const sortedTrackers = computed(() =>
+  sortTrackers(filteredTrackers.value)
+);
+
+type sortKeys = keyof Pick<ZodTrackerType,
+  'created_dt' | 'updated_dt'
+>;
+
+const sortKey = ref<sortKeys>('created_dt');
+const sortOrder = ref(ascend);
+
+const setSort = (key: sortKeys) => {
+  if (sortKey.value === key) {
+    sortOrder.value = sortOrder.value === ascend ? descend : ascend;
+  } else {
+    sortKey.value = key;
+    sortOrder.value = ascend;
+  }
+};
+
+function sortTrackers(trackers: TrackerWithModule[]): TrackerWithModule[] {
+  const customSortKey = sortKey.value;
+  const order = sortOrder.value;
+
+  const customSortFn = (affect: TrackerWithModule) => {
+    return affect[customSortKey] || 0;
+  };
+
+  const comparator = [order<TrackerWithModule>(customSortFn)];
+
+  return sortWith([
+    ...comparator
+  ])(trackers);
+}
+
+// Trackers filters by field
+const filteredTrackers = computed(() => {
+  return props.displayedTrackers.filter(tracker => {
+    const matchesStatusFilter =
+      statusFilter.value.length === 0 || statusFilter.value.includes(tracker.status ?? 'EMPTY');
+    return matchesStatusFilter;
+  });
+});
+
+const trackerStatuses = computed(() => {
+  const statuses = props.displayedTrackers.map(item => item.status?.toUpperCase() || 'EMPTY');
+  const uniqueStatuses = [...new Set(statuses)];
+  return uniqueStatuses;
+});
+
+const statusFilter = ref<string[]>([]);
+
+function toggleFilter(filterArray: Ref<string[]>, item: string) {
+  const index = filterArray.value.indexOf(item);
+  if (index > -1) {
+    filterArray.value.splice(index, 1);
+  } else {
+    filterArray.value.push(item);
+  }
+}
+
+function toggleStatusFilter(status: string) {
+  toggleFilter(statusFilter, status);
+}
+
+// Trackers pagination
+const totalPages = computed(() =>
+  Math.ceil(sortedTrackers.value.length / settings.value.trackersPerPage)
+);
+
+const minItemsPerPage = 5;
+const maxItemsPerPage = 20;
+const {
+  pages,
+  itemsPerPage,
+  currentPage,
+  changePage,
+} = usePagination(totalPages, settings.value.trackersPerPage);
+
+const paginatedTrackers = computed(() => {
+  const start = (currentPage.value - 1) * settings.value.trackersPerPage;
+  const end = start + settings.value.trackersPerPage;
+  return sortedTrackers.value.slice(start, end);
+});
+
+function reduceItemsPerPage() {
+  if (settings.value.trackersPerPage > minItemsPerPage) {
+    settings.value.trackersPerPage--;
```

Comment:
Mutating a pinia store should be avoided, use the `saveSettings` method or pinia `$patch`
Another option could be to implement `increment` and `decrease` methods